### ISSUE TITLE
Browse local files through chat commands

### ISSUE THREAD

**Issue by Jokler** [hivecom/pokebot#80]
**Is your feature request related to a problem? Please describe.**
Currently it is not possible to see what files are available through the filesystem.

**Describe the solution you'd like**
A chat command to list files or search them. Listing them could be similar to OS style commands(e.g. ls, cd).

**Describe alternatives you've considered**
Files could also be discovered through the web interface, though I think that should be a separate feature on its own.

**Comment by TheGreatestJannet** [comment]
Thanks a lot for this feature! I have just tried it and it works very well! However, there is no way to tell if the song it is playing is local or if it used YouTube. I wonder if something could be added to the response such as `Playing local song Mr Blue Sky - Electric Light Orchestra` or something similar.

I have just noticed that online songs have the length and local songs don't but that's a rather obscure method of telling if it is local or not.
